fastai/vision/augment.py

```diff
@@ -259,14 +278,24 @@ def encodes(self, x:(Image.Image,TensorBBox,TensorPoint)):
 class RandomResizedCrop(RandTransform):
     "Picks a random scaled crop of an image and resize it to `size`"
     split_idx,order = None,1
-    def __init__(self, size, min_scale=0.08, ratio=(3/4, 4/3), resamples=(Image.BILINEAR, Image.NEAREST),
-                 val_xtra=0.14, max_scale=1., **kwargs):
+    def __init__(self,
+         size:(int, tuple), # Final size, duplicated if one value is specified,,
+         min_scale:float=0.08, # Minimum scale of the crop
+         ratio:(float, float)=(3/4, 4/3), # Range of width over height of the output
+         resamples=(Image.BILINEAR, Image.NEAREST), # Pillow `Image` resample mode, resamples[1] for mask
+         val_xtra:float=0.14, # Ratio of extra pixels in the validation set
+         max_scale:float=1., # Maximum scale of the crop
```

Comment:
```suggestion
         min_scale:float=0.08, # Minimum scale of the crop, in relation to image area
         ratio:(float, float)=(3/4, 4/3), # Range of width over height of the output
         resamples=(Image.BILINEAR, Image.NEAREST), # Pillow `Image` resample mode, resamples[1] for mask
         val_xtra:float=0.14, # For validation set, ratio of size extra pixels to use in a squish before cropping to size.
         max_scale:float=1., # Maximum scale of the crop, in relation to image area
```

Reply:
For the val_xtra, I don't quite understand why we need to squish to a bigger dimension then crop down to size? It just means we cut the pixels at the border

Reply:
I think the model is trained with crop image then it is better if the validation image is going through some sort of cropping too.

For the document, How about,  `the ratio of size at the edge cut out in the validation set`? 

Thanks

Reply:
First comment: Because depending on the squish algorithm it needs to have some extra border so that is can get similar resizing artifacts to that which we see during training. We resize to be a bit bigger than our final crop so that most of the image will be included in the crop. We want a large center crop during validation so that the model can see most of the image at test time. 

Second comment: For validation we resize to approximately:             
```
xtra =largest_img_deminsion * self.val_xtra
self.final_size = (self.size[0]+xtra, self.size[1]+xtra)
```
First for validation we resize to self.final_size. We then crop to the regular `size` parameter. For training the transform randomly does the resize, and crop. For validation this is just an easier version that means our model will have an easier time at validation/test time, giving us better results in production in the end. 

`the ratio of size at the edge cropped out in the validation set`
switch cut for cropped and we have a winner.